packages/component-library/components/Dropdown/Dropdown.module.scss

```diff
@@ -1,98 +0,0 @@
-@import "~@kaizen/design-tokens/sass/color";
```

Comment:
If we're not doing a breaking change on this PR, we'll want to delete these in a separate PR and trigger a breaking change for component-library.

There's about 7 usages of the Dropdown component in Murmur.

Reply:
https://github.com/cultureamp/kaizen-design-system/pull/2739